packages/synapse-react-client/src/components/SynapseTable/TopLevelControls/TopLevelControlsUtils.ts

```diff
@@ -17,9 +18,9 @@ const TO_DOWNLOAD_CART = 'to Download Cart'
  * @param data
  */
 export function getNumberOfResultsToInvokeAction(
-  hasSelectedRows: QueryContextType['hasSelectedRows'],
-  selectedRows: QueryContextType['selectedRows'],
-  data: QueryContextType['data'],
```

Comment:
These are going to change when this data is removed from the context object, so update the types now.